### PR TITLE
Adding Tailscale to Web Services

### PR DESCRIPTION
**Pull request by cgimenes**
Internal: https://github.com/MustardOS/internal/pull/382

### CODE REVIEW

**Comment by cgimenes** [comment]
Internal: https://github.com/MustardOS/internal/pull/382

File: muxwebserv/main.c [hunk 1]
```diff
@@ -239,7 +256,7 @@ void init_navigation_groups() {
     apply_theme_list_drop_down(&theme, ui_droTTYD, options);
     apply_theme_list_drop_down(&theme, ui_droSyncthing, options);
     apply_theme_list_drop_down(&theme, ui_droRSLSync, options);
-    apply_theme_list_drop_down(&theme, ui_droNTP, options);
+    apply_theme_list_drop_down(&theme, ui_droTailscaled, options);
```
Comment (xonglebongle) [inline muxwebserv/main.c:259]: Whoopsies! I think you meant to add to the drop down list and not remove NTP haha!

Comment (cgimenes) [reply muxwebserv/main.c:259]: That's right! Haha

**Comment by xonglebongle** [comment]
@cgimenes could you also let me know if you are part of the muOS Discord Community what your handle is so I can adjust the contributors file.

**Comment by cgimenes** [comment]
> @cgimenes could you also let me know if you are part of the muOS Discord Community what your handle is so I can adjust the contributors file.

Unfortunately, I don't use Discord. But you can use my Github nick if it's ok for you.

BTW thank you for accepting my PR.

**Comment by xonglebongle** [comment]
Well if you ever change your mind you have a permanent testing role for updates!

**Comment by cgimenes** [comment]
Cool! Try this one `mardeoli`

**Pull request merged**